src/themes/latex/styles.css

```
@@ -0,0 +1,42 @@
+@import '../../extensions/cite-apa/styles.css';
+@import '../../extensions/code/styles.css';
+@import '../../extensions/math/styles.css';
+@import '../../extensions/pages/styles.css';
+@import '../../extensions/person/styles.css';
+
+@import 'https://raw.githubusercontent.com/davidrzs/latexcss/9264a50c252d47fbe6d4998dbc5581882d79ab3c/latex.css';
+
+:--Article:--root {
+  --spacer-medium: 1em;
+  --spacer-large: 10em;
```

An optional, but nice to have, set of variables to set here would be for the `font-family-x` ones so that they get applied to the web component elements such as buttons and tooltips.

```suggestion
  --spacer-large: 10em;
  --font-family-display: Latin Modern Roman, serif;
  --font-family-body: Latin Modern Roman, serif;
```